### PR TITLE
Update erc-777 developer docs (French)

### PR DESCRIPTION
**Pull request by AsheBarrett**
Some sentences did not the right translation due to automatic translation and was confusing.

### CODE REVIEW

File: src/content/translations/fr/developers/docs/standards/tokens/erc-777/index.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ L’ERC-777 apporte les améliorations suivantes par rapport à l’ERC-20 :
...
 
-Les crochets sont enregistrés et découverts en utilisant la norme [ERC-1820](https://eips.ethereum.org/EIPS/eip-1820).
+Les crochets sont enregistrés et accessibles en utilisant la norme [ERC-1820](https://eips.ethereum.org/EIPS/eip-1820).
 
 #### Pourquoi les crochets sont-ils exceptionnels ? {#why-are-hooks-great}
 
 1. Les crochets permettent d'envoyer des jetons à un contrat et de notifier le contrat en une seule transaction, contrairement à [ERC-20](https://eips.ethereum.org/EIPS/eip-20), qui nécessite un double appel (`approve`/`transferFrom`) pour y parvenir.
-2. Les contrats qui n'ont pas de crochets enregistrés sont incompatibles avec l'ERC-777. Le contrat envoyé annulera la transaction lorsque le contrat de réception n'a pas enregistré de crochet. Cela empêche les transferts accidentels vers des contrats intelligents non ERC-777.
+2. Les contrats qui n'ont pas implémentés les crochets sont incompatibles avec l'ERC-777. Le contrat envoyé annulera la transaction lorsque le contrat de réception n'a pas implémenté de crochet. Cela empêche les transferts accidentels vers des contrats intelligents non ERC-777.
```
Comment (vdusart) [inline src/content/translations/fr/developers/docs/standards/tokens/erc-777/index.md:28]:
```suggestion
2. Les contrats qui n'ont pas implémenté les crochets sont incompatibles avec l'ERC-777. Le contrat envoyé annulera la transaction lorsque le contrat de réception n'a pas implémenté de crochet. Cela empêche les transferts accidentels vers des contrats intelligents non ERC-777.
```

**Review by vdusart — CHANGES_REQUESTED** [review]

File: src/content/translations/fr/developers/docs/standards/tokens/erc-777/index.md (hunk 1 above)
Comment (minimalsm) [reply src/content/translations/fr/developers/docs/standards/tokens/erc-777/index.md:28]: Ty @vdusart

**Review by minimalsm — APPROVED** [review]
Thanks @AsheBarrett 🚢

**Comment by minimalsm** [comment]
@all-contributors please add @AsheBarrett and @vdusart for translations

**Pull request merged**